Demo/AuthenticatorDemo/ViewController+WordPressAuthenticator.swift

```diff
@@ -96,6 +97,12 @@ extension ViewController {
                 )
             } catch let error as OAuthError {
                 presentAlert(title: "❌", message: error.errorDescription, onDismiss: {})
+            } catch let error as NSError where
+                error.domain == ASWebAuthenticationSessionError.errorDomain
+                && error.code == ASWebAuthenticationSessionError.Code.canceledLogin.rawValue {
```

Comment:
I you ask Xcode, it will indent this as:

```swift
            } catch let error as NSError where
                        error.domain == ASWebAuthenticationSessionError.errorDomain
                        && error.code == ASWebAuthenticationSessionError.Code.canceledLogin.rawValue {
```

That doesn't make any sense to me, so I updated it manually.